CaesarCipher.java

```diff
@@ -0,0 +1,49 @@
+import java.util.Scanner;
+
+/*
+	https://www.hackerrank.com/challenges/caesar-cipher-1
+ */
+
+public class CaesarCipher {
+
+	public static int forUpper(int ascii,int K)
+	{
+		if(ascii + K > 90)
+			return forUpper(ascii - 26, K);
+		else 
+			return ascii + K;
+	}
+	public static int forLower(int ascii,int K)
+	{
+		if(ascii + K > 122)
+			return forLower(ascii - 26, K);
+		else 
+			return ascii + K;
+	}
+	public static void main(String[] args) {
+		
+		Scanner s = new Scanner(System.in);
+		int N = s.nextInt();
+		String S = s.next();
+		int K = s.nextInt();
+		int ascii=0;
+		String Encrypted = "";
+		
+		for(int index = 0; index < N; index++)
+		{
+			ascii = (int)S.charAt(index);
+			if(ascii >= 65 && ascii <= 90)

```

Comment:
Instead of 2 functions for lower and upper case, you can use just one and a modulus `%` operator.
Your code looks fine. Here is an alternative approach with one function for your reference:

```Java
int roundIncrement(int ascii, int increment, boolean isLower){
    int offset = isLower ? 97: 65;
    ascii -= offset; // now ascii is between 0 and 25
    ascii += increment;
    return offset + (ascii % 26); // trim down and offset
}
```
Inside for loop:

```Java
boolean isLower = false;
ascii = (int)S.charAt(index);
if(ascii >= 65 && ascii <= 90){
    ascii = roundIncrement(ascii, K, false);
}else if(ascii >= 97 && ascii <= 122){
     ascii = roundIncrement(ascii, K, true);
}
```